lib/acl-checker.js

```diff
@@ -20,7 +22,7 @@ class ACLChecker {
     var accessType = 'accessTo'
     var possibleACLs = ACLChecker.possibleACLs(resource, this.suffix)
     // If this is an ACL, Control mode must be present for any operations
-    if (this.isAcl(resource)) {
+    if (this.isAcl(resource, this.suffix)) {
```

Comment:
Can this.suffix not be used directly in `this.isAcl`?
In the rest of the code `this.isAcl` doesn't take any second parameter